Game/Config/DefaultSpatialGDKEditorSettings.ini

```diff
@@ -2,4 +2,7 @@
 
 [/Script/SpatialGDKEditor.SpatialGDKEditorSettings]
 LaunchConfigDesc=(Template="w2_r0500_e5",World=(Dimensions=(X=10000,Y=10000),ChunkEdgeLengthMeters=50,SnapshotWritePeriodSeconds=0,LegacyFlags=(("bridge_qos_max_timeout", "0"),("bridge_soft_handover_enabled", "false"),("bridge_single_port_max_heartbeat_timeout_ms", "3600000")),LegacyJavaParams=()),ServerWorkers=((WorkerTypeName="UnrealWorker"),(WorkerTypeName="NPCWorker",NumEditorInstances=0)))
+StandardRuntimeVersion=(bUseGDKPinnedRuntimeVersionForLocal=True,bUseGDKPinnedRuntimeVersionForCloud=True,LocalRuntimeVersion="",CloudRuntimeVersion="")
+bDeleteDynamicEntities=True
+AutoStopLocalDeployment=OnEndPIE
```

Comment:
Should this be default in the GDK now?

Reply:
Yes - although it can be done with the workflow changes @joshuahuburn is working on